WebApiCoreSeed.WebApi.Tests/Controllers/UserControllerTest.cs

```diff
@@ -0,0 +1,260 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Moq;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using WebApiCoreSeed.Data.Models;
+using WebApiCoreSeed.Domain.Services.Interfaces;
+using WebApiCoreSeed.WebApi.Controllers;
+using WebApiCoreSeed.WebApi.Models;
+using Xunit;
+
+namespace WebApiCoreSeed.WebApi.Tests.Controllers
+{
+    public class UserControllerTest
+    {
+        #region Fields
+
+        private Mock<IUserService> _userService;
+
+        private UserController _classUnderTest;
+
+        #endregion
+
+        #region Constructors
+
+        public UserControllerTest()
+        {
+            _userService = new Mock<IUserService>();
+            _classUnderTest = new UserController(_userService.Object);
```

Comment:
Let's create a pristine instance per test, instead of reusing the same one for all.